plover/gui/config.py

```diff
@@ -200,20 +202,30 @@ def __init__(self, config, parent):
         sizer = wx.FlexGridSizer(2, 3)
         sizer.AddGrowableCol(1)
 
-        sizer_flags = wx.SizerFlags(1).Align(wx.ALIGN_CENTER_VERTICAL).Border(wx.ALL, UI_BORDER)
+        sizer_flags = wx.SizerFlags(1) \
+            .Align(wx.ALIGN_CENTER_VERTICAL) \
+            .Border(wx.ALL, UI_BORDER)
 
-        sizer.AddF(wx.StaticText(self, label=MACHINE_LABEL), sizer_flags.Left())
+        sizer.AddF(
```

Comment:
And here I'd just break after the comma,

```
name.call(parameter1,
          parameter2)
```

Again, just going off of intuition, not sure if that is written about.